src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -88,75 +146,168 @@ public static class ArmConstants { // To find values later
 	private final ProfiledPIDController armPID;
 	private final ProfiledPIDController wristPID;
 
+	private final ArmFeedforward wristFeedforward;
+
 	// Constructor
 
 	public ArmSubsystem() {
-		armMotor = new CANSparkMax(ARM_MOTOR, MotorType.kBrushless);
+		armMotor1 = new CANSparkMax(ARM_MOTOR1, MotorType.kBrushless);
+		armMotor2 = new CANSparkMax(ARM_MOTOR2, MotorType.kBrushless);
 		wristMotor = new CANSparkMax(WRIST_MOTOR, MotorType.kBrushless);
+
 		shoulderEncoder = new Encoder(SHOULDER_ENCODER_PORT_A, SHOULDER_ENCODER_PORT_B);
 		wristEncoder = new Encoder(WRIST_ENCODER_PORT_A, WRIST_ENCODER_PORT_B);
 
 		armPID = new ProfiledPIDController(ARM_K_P, ARM_K_I, ARM_K_D, ARM_CONSTRAINTS);
 		wristPID = new ProfiledPIDController(WRIST_K_P, WRIST_K_I, WRIST_K_D, WRIST_CONSTRAINTS);
 
-		armMotor.setIdleMode(IdleMode.kBrake);
+		wristFeedforward = new ArmFeedforward(WRIST_K_A, WRIST_K_G, WRIST_K_S, WRIST_K_V);
+
+		armMotor2.follow(armMotor1, true);
+
+		armMotor1.setIdleMode(IdleMode.kBrake);
+		armMotor2.setIdleMode(IdleMode.kBrake);
 		wristMotor.setIdleMode(IdleMode.kBrake);
+
 		armPID.setTolerance(ARM_POS_TOLERANCE, ARM_VELOCITY_TOLERANCE);
 		wristPID.setTolerance(WRIST_POS_TOLERANCE, WRIST_VELOCITY_TOLERANCE);
 
+		currentPosition = ARM_LOW_POSITION;
```

Comment:
Seems like this needs to come from the through bore encoders and not hardcoded if the robot is ever started when the arm is not all the way down.